scripts/install

```diff
@@ -116,7 +118,7 @@ ynh_system_user_create --username=$app
 ynh_script_progression --message="Configuring php-fpm..." --weight=2
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config --usage=medium --footprint=high
+ynh_add_fpm_config --phpversion="$php_version"
```

Comment:
Both options are compatible. Why not keep also the adaptative config for php ?

Reply:
OK, will fix that.